src/popup/App.vue

```diff
@@ -107,6 +108,7 @@ export default defineComponent({
     const { addWalletNotification } = useNotifications({ store });
     const { loadCoinsData } = useCurrencies({ withoutPolling: true });
     const { initViewport } = useViewport();
+    useFungibleTokens({ store });
```

Comment:
We need to choose one of two options:
1. Fetch fungible tokens only if they are needed
2. Poll them all the time

With this line it is this weird mix of those two. The composable is using polling based on mounted components, which means we want to poll only when needed. But here we are loading the composable globally so the polling will be happening all the time as there is always at least one component that is using the composable.

My suggestion: remove this line.